src/OpenTelemetry.Contrib.Instrumentation.Wcf/README.md

```diff
@@ -20,6 +20,29 @@ using var openTelemetry = Sdk.CreateTracerProviderBuilder()
     .Build();
 ```
 
+Instrumentation can be configured via options overload for 
+`AddWcfInstrumentation` method:
+
+```csharp
+using var openTelemetry = Sdk.CreateTracerProviderBuilder()
+    .AddWcfInstrumentation(options => 
+    {
+        options.SuppressDownstreamInstrumentation = false;  
+
+        // Enable enriching an activity after it is created.
+        options.Enrich = (activity,eventName,message) =>    
+        {
+            // For event names, please refer to string constants in 
+            // WcfEnrichEventNames class.
+            if(eventName == WcfEnrichEventNames.AfterReceiveRequest)  
+            {
+                activity.AddTag("companyname.customtag", Guid.NewGuid());
```

Comment:
```suggestion
                activity.SetTag("companyname.customtag", Guid.NewGuid());
```

Reply:
Guid is not a supported type for span/activity attributes/tags.

Reply:
Code example changed, see comment below.